### ISSUE TITLE
Allow user to specify contents of main.cpp manually.

### ISSUE THREAD

**Issue by mileshiroo** [olab-io/ofSketch#139]
Hello,

I'm trying to use run shaders with the ofShader object, which requires this call before ofSetupOpenGL(1024, 768, OF_WINDOW) in main(): 

```
ofSetCurrentRenderer(ofGLProgrammableRenderer::TYPE);
```

Is there any way to do this in ofSketch, that is, is there a way to modify what goes in main()? Running the line in setup() creates a second window. Thank you!

**Comment by bakercp** [comment]
Excellent question.  We're currently working on handlers so that you can just use the standard main.cpp / ofApp.h/cpp, etc.  That will make it easier.  In the mean time, you could modify Resources/Templates/CompilerTemplates/main.tmpl ... not ideal though.

**Comment by brannondorsey** [comment]
Yes, this is not ideal at the moment. Additionally, you may need to make sure that ofSketch is closed when making an edit to `main.cpp` so that the changes you make are correctly loaded into the app.

**Comment by mileshiroo** [comment]
Great, thanks for the response @bakercp  and @brannondorsey!
